events/handler_test.go

```diff
@@ -48,7 +48,8 @@ func initializeScenario(t *testing.T) (*events, *Request, *info.Event, *info.Eve
 	fakeEvent := makeEvent(createOldTime(t), "/")
 	fakeEvent2 := makeEvent(time.Now(), "/")
 
-	return NewEventManager(DefaultStoragePolicy()), NewRequest(), fakeEvent, fakeEvent2
+	manager := NewEventManager(DefaultStoragePolicy())
+	return manager.(*events), NewRequest(), fakeEvent, fakeEvent2
```

Comment:
can we change the return type to EventManager instead of casting, or would that break the test?

Reply:
I would have broken it as one of the tests relied on internal structures of `events` instead of using interface methods. Fixed :)